configs/eslint-config-compass/index.js

```diff
@@ -24,39 +25,46 @@ const reactConfigurations = [
 ];
 
 const testConfigurations = ['plugin:mocha/recommended'];
+
 const testRules = {
   'mocha/no-exclusive-tests': 'error',
   'mocha/no-hooks-for-single-case': 'off',
+  'mocha/no-setup-in-describe': 'off',
   '@typescript-eslint/no-explicit-any': 'off',
   '@typescript-eslint/no-empty-function': 'off',
 };
 
 module.exports = {
   plugins: ['@typescript-eslint', 'jsx-a11y', 'mocha', 'react', 'react-hooks'],
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    ecmaVersion: 'latest',
+    requireConfigFile: false,
+    babelOptions: {
+      presets: [
+        require.resolve('@babel/preset-env'),
+        require.resolve('@babel/preset-react'),
+      ],
+    },
+  },
```

Comment:
Before we were running eslint only against javascript that was parseable by their built-in parser, not a case anymore